Library/Formula/qprint.rb

```diff
@@ -1,14 +1,19 @@
-require 'formula'
-
 class Qprint < Formula
-  homepage 'http://www.fourmilab.ch/webtools/qprint'
-  url 'http://www.fourmilab.ch/webtools/qprint/qprint-1.0.tar.gz'
-  sha1 '533a4942e93cccc2e6b3fd2171707bf1d0054d20'
+  homepage "http://www.fourmilab.ch/webtools/qprint"
```

Comment:
Can you shunt this and the `url` to `https://` please? Upstream seem to support it. Thanks!